docs/user_guide/usergroup_resolution.md

```diff
@@ -86,14 +86,16 @@ metadata:
   annotations:
     yunikorn.apache.org/user.info: "
     {
-      username: \"yunikorn\",
-      groups: [
+      \"user\": \"yunikorn\",
+      \"groups\": [
         \"developers\",
         \"devops\"
       ]
     }"
 ```
 
+(NOTE: for historical reason, "username" is used in label "yunikorn.apache.org/username", whereas, "user" is used in the annotation here)
```

Comment:
I think we should leave the note out as it's completely obsolete. It's likely to cause more confusion than not. The old label still exists but has completely different meaning and semantics.